src/bramble/client/StateManager.js

```diff
@@ -103,8 +116,8 @@ define(function() {
                 set: function(v) { storage.setItem(prefix("wordWrap"), v); }
             },
             autoCloseTags: {
-                get: function()  { return getBool(storage, "closeTags"); },
-                set: function(v) { storage.setItem(prefix("closeTags"), v); }
+                get: function()  { return getObject(storage, "closeTags"); },
+                set: function(v) { storage.setItem(prefix("closeTags"), JSON.stringify(v)); }
```

Comment:
Interesting, yeah I guess we should probably make sure we handle cyclic references!

@cgsingh would you be able to add this method near `getObject`:
```js
function setObject(storage, property, obj) {
  var objStr;
  property = prefix(property);

  try {
    objStr = JSON.stringify(obj);
    storage.setItem(property, objStr);
  } catch(e) {
    console.error("Failed to stringify object to write to localStorage for item " + property + " with: ", e);
  }
}
```
and call this function here?